templates/repo/header_icon.tmpl

```diff
@@ -9,7 +9,10 @@
 		{{svg "octicon-lock" 32}}
 	{{else if and (not $.IsMirror) (not $.IsFork) ($.Owner)}}
 		{{if $.Owner.Visibility.IsPrivate}}
-			{{svg "octicon-internal-repo" 32}}
+			<div class="repo-owner-avatar">
```

Comment:
Maybe move this to top-level in this file and name it `repo-header-icon`, just in case we want to add more similar "sub-icons" later.